package.json

```diff
@@ -8,7 +8,7 @@
   "private": true,
   "scripts": {
     "bootstrap": "lerna bootstrap",
-    "build": "yarn clean-build && lerna run build --concurrency 1 --ignore @data-driven-forms/react-renderer-demo --ignore @data-driven-forms/pf3-component-mapper",
+    "build": "yarn clean-build && lerna run build --concurrency 1 --ignore @data-driven-forms/react-renderer-demo",
```

Comment:
We should ignore the PF3 mapper. The build was not adjusted and we won't continue with development.

Reply:
there is no pf3 mapper

Reply:
Oh the package is gone? I forgot.

Reply:
![image](https://user-images.githubusercontent.com/32869456/109009771-3bd37400-76af-11eb-940d-0727d264beb5.png)